docs/src/app/versions.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
And regardless - while navigating around old versions would be a bonus, in reality if you're using a particular version, you'll navigate to it and stick with it, so I don't think getting this working is a priority for 0.15.0. Having the old version docs available should be enough.